<launch>

<arg name="cam0_name" default="cam0"/>
<!-- <arg name="cam0_dev" value="/dev/video2"/> -->
<arg name="cam0_dev" value="/dev/v4l/by-id/usb-Linux_Foundation_Webcam_gadget_0123-video-index0"/>

<group ns="aruco0">
  <node name="$(arg cam0_name)" pkg="usb_cam" type="usb_cam_node" output="screen" respawn="true">
    <param name="video_device" value="$(arg cam0_dev)"/>
    <param name="image_width" value="1920" />
    <param name="image_height" value="1080" />
    <param name="pixel_format" value="mjpeg" />
    <param name="color_format" value="yuv420p" />
    <param name="framerate" value="30" />
    <param name="camera_frame_id" value="$(arg cam0_name)" />
    <param name="io_method" value="mmap"/>
    <param name="camera_info_url" value="package://sisyph_slam/config/aceline_s200_1080p_30fps.yaml"/>
  </node>
</group>

</launch>
